prost-derive/src/field/mod.rs

```diff
@@ -86,6 +91,7 @@ impl Field {
             Field::Map(ref map) => vec![map.tag],
             Field::Oneof(ref oneof) => oneof.tags.clone(),
             Field::Group(ref group) => vec![group.tag],
+            Field::Unknown(_) => vec![],
```

Comment:
Is this correct? The unknown fields have tags, right? I have not looked into what this function is used for.